Dockerfile

```diff
@@ -19,6 +19,9 @@ RUN apt-get update && apt-get install -y \
     groupadd -r bee --gid 999; \
     useradd -r -g bee --uid 999 --no-log-init -m bee;
 
+# make sure mounted volumes have correct permissions
+RUN mkdir -p /home/bee/.bee && chown 999:999 /home/bee/.bee
```

Comment:
If folder is not precreated and permissions set when you mount external volume it will be owned by root